modules/ROOT/pages/servicenow/servicenow-reference.adoc

```diff
@@ -3,107 +3,124 @@ ifndef::env-site,env-github[]
-** JAKARTA |  |  JAKARTA |
-| User Table List a| Array of String |  |  |
+** JAKARTA
+** KINGSTON
+** MADRID |  |  +++MADRID+++ | 
+| User Table List a| Array of String |  |  | 
 | Transport Configuration a| One of:
-* BasicAuthHttpMessageDispatcherProvider
-* HttpRequesterBasedTransportProvider |  |  | x
-| Reconnection a| Reconnection |  When the application is deployed, a connectivity test is performed on all connectors. If set to true, deployment fails if the test doesn't pass after exhausting the associated reconnection strategy. |  |
-| Pooling Profile a| PoolingProfile |  Characteristics of the connection pool. |  |
+
+* <<BasicAuthHttpMessageDispatcherProvider>>
+* <<HttpRequesterBasedTransportProvider>> |  |  | x
+| Reconnection a| <<Reconnection>> |  +++When the application is deployed, a connectivity test is performed on all connectors. If set to true, deployment fails if the test doesn't pass after exhausting the associated reconnection strategy+++ |  | 
+| Pooling Profile a| <<PoolingProfile>> |  +++Characteristics of the connection pool+++ |  | 
 |===
 
-=== Associated Operations
+== Supported Operations
+* <<invoke>> 
 
-* invoke
 
-== Invoke Operation
 
-`<servicenow:invoke>`
+== Operations
 
-Invokes Web Service operations.
+[[invoke]]
+=== Invoke
+`<servicenow:invoke>`
 
++++
+invokes Web Service operations
++++
 
+==== Parameters
 [%header%autowidth.spread]
 |===
 | Name | Type | Description | Default Value | Required
 | Configuration | String | The name of the configuration to use. | | x
 | Service a| String |  |  | x
 | Operation a| String |  |  | x
-| Body a| Binary |  The XML body to include in the SOAP message, with all the required parameters. |  `#[payload]` |
-| Headers a| Binary |  The XML headers to include in the SOAP message. |  |
-| Attachments a| Object |  The attachments to include in the SOAP request. |  |
-| Headers a| Object |  The headers to set in the transport configuration. |  |
-| Target Variable a| String |  The name of a variable on which the operation's output is placed. |  |
-| Target Value a| String |  An expression that is evaluated against the operation's output and the outcome of that expression is stored in the target variable. |  `#[payload]` |
-| Reconnection Strategy a| * reconnect
-* reconnect-forever |  A retry strategy in case of connectivity errors. |  |
+| Body a| Binary |  +++The XML body to include in the SOAP message, with all the required parameters.+++ |  `#[payload]` | 
+| Headers a| Binary |  +++The XML headers to include in the SOAP message.+++ |  | 
+| Attachments a| Object |  +++The attachments to include in the SOAP request.+++ |  | 
+| Headers a| Object |  +++The headers to set in the transport configuration.+++ |  | 
+| Target Variable a| String |  +++The name of a variable to store operation's output.+++ |  | 
+| Target Value a| String |  +++An expression to evaluate against the operation's output and store the expression outcome in the target variable+++ |  `#[payload]` | 
```

Comment:
An expression that evaluates against the operation's output and stores the outcome in the target variable.